tests/features/solution/solution.edit.feature

```diff
@@ -84,7 +82,7 @@ Feature: Solution editing.
     Then I delete the "Solution A revised" solution
 
   Scenario: A solution facilitator can edit only the solutions he's associated with.
-    Given the following solution:
+    And the following solution:
```

Comment:
Hmm this change was unintentional, but I am now afraid to fix it and hit random test bot failures again :(